src/views/OurTeamPageView/TeamMemberCard/TeamMemberCard.styles.js

```diff
@@ -14,6 +14,20 @@ import { breakpoint } from '../TeamGallery/TeamGallery.styles';
 import { speed } from '../../../styles/animation';
 
 export const galleryStyleCard = css`
+  @media screen and (prefers-reduced-motion: reduce) {
+    &:hover,
+    &:focus {
+      transition: none;
+    }
+  }
+
+  @media screen and (prefers-reduced-motion: reduce) {
```

Comment:
Apparaît deux fois ici
```suggestion

```

Reply:
Préférable de mettre `@media screen and (prefers-reduced-motion: reduce)` après 
```suggestion
  &:hover,
  &:focus {
    transition: transform ${speed.fast};
  }

  @media screen and (prefers-reduced-motion: reduce) {
    &:hover,
    &:focus {
      transition: none;
    }
  }
 
  // ou plus simplement
  @media screen and (prefers-reduced-motion: no-preference) {
    &:hover,
    &:focus {
      transition: transform ${speed.fast};
    }
  }
```